examples/Rudaina_work_area/Type-Based Directory/genzsprtwo_27_dir_orgnizer.py

```diff
@@ -0,0 +1,28 @@
+import os
+import shutil
+import glob
+
+DIRECTORY = "./dir"
+
+
+def dir_organizer(source_dir):
```

Comment:
I believe this is creating directories instead of organizing it so we can name it something like, create_dirs_if_not_exists